Gemfile

```diff
@@ -8,7 +8,7 @@ gemspec
 gem "appbundler"
 gem "chef-config", git: "https://github.com/chef/chef", branch: "main", glob: "chef-config/chef-config.gemspec"
 gem "chef-utils", git: "https://github.com/chef/chef", branch: "main", glob: "chef-utils/chef-utils.gemspec"
-gem "ffi", "~> 1.17", force_ruby_platform: true
+gem "ffi", "<= 1.17", force_ruby_platform: true
```

Comment:
@johnmccrae is this breaking somewhere? @jaymzh was making a major effort to get `ffi` up-to-date so that we're not caught flat-footed when things need the newer version.

Reply:
I undid this and it works fine